spec/services/marketing_site_spec.rb

```diff
@@ -29,6 +29,22 @@
     end
   end
 
+  describe '.messaging_practices_url' do
+    it 'points to messaging practices section of the privacy page' do
+      expect(MarketingSite.messaging_practices_url).to \
```

Comment:
Our line length and code style can a bit tricky, the most common way to make all the rules happy to break a line is to move the `to` onto the next line:

```ruby
expect(MarketingSite.messaging_practices_url).
  to eq('https://www.login.gov/policy/#our-messaging-practices')
```